src/components/TextInput/BaseTextInput.js

```diff
@@ -343,7 +343,7 @@ class BaseTextInput extends Component {
                                 />
                                 {Boolean(this.props.secureTextEntry) && (
                                     <Checkbox
```

Comment:
Don't we need to pass `accessibilityLabel` here?

Reply:
Yes, I just add it here and in other components